components/auth/LoginMenu.tsx

```diff
@@ -11,6 +11,7 @@ type LoginMenuProp = {
 const LoginMenu = ({ isAuthenticated }: LoginMenuProp) => {
   const { i18n, t } = useTranslation("common");
   const handleClick = () => {
+    sessionStorage.clear();
```

Comment:
This could cause issues --- if someone has started a form but didn't sign in.... once they login the form will be gone.

Reply:
I don't think so @timarney . The handle click only gets called for the sign out button no ?

Reply:
You're right mis-read on my first pass of this PR.